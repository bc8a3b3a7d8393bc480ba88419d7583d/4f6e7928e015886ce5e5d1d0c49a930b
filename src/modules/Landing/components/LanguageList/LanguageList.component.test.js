import { testSnapshots } from '../../../../utils/test.util';

import Component from './LanguageList.component';

describe('Modules/Landing - Language List', () => {
  const componentProps = {
    onSelectLanguage: jest.fn(),
  };

  testSnapshots(Component, [
    {
      description: 'basic render',
      props: componentProps,
    },
  ]);
});
