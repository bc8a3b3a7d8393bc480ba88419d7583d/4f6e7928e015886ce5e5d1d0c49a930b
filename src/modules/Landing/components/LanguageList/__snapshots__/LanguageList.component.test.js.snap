// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules/Landing - Language List snapshots basic render 1`] = `
<Component>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontFamily": "Oswald-Regular",
        "fontSize": 12,
        "textAlign": "center",
      }
    }
  >
    landing.content.selectLanguage
  </Text>
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="VN"
    countryName="Việt Nam"
    key="VN-vi"
    languageCode="vi"
    languageName="Tiếng Việt"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="VN"
    countryName="Viet Nam"
    key="VN-en"
    languageCode="en"
    languageName="English"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="ID"
    countryName="Indonesia"
    key="ID-en"
    languageCode="en"
    languageName="English"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="ID"
    countryName="Indonesia"
    key="ID-id"
    languageCode="id"
    languageName="Indonesia"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="MY"
    countryName="Malaysia"
    key="MY-ms"
    languageCode="ms"
    languageName="Bahasa Melayu"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="MY"
    countryName="Malaysia"
    key="MY-en"
    languageCode="en"
    languageName="English"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="PH"
    countryName="Philipines"
    key="PH-en"
    languageCode="en"
    languageName="English"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="SG"
    countryName="Singapore"
    key="SG-en"
    languageCode="en"
    languageName="English"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="TH"
    countryName="Thailand"
    key="TH-en"
    languageCode="en"
    languageName="English"
    onPress={[MockFunction]}
    selected={false}
  />
  <LanguageItem
    containerStyle={
      Object {
        "marginTop": 5,
      }
    }
    countryCode="TH"
    countryName="Thailand"
    key="TH-th"
    languageCode="th"
    languageName="ไทย"
    onPress={[MockFunction]}
    selected={false}
  />
</Component>
`;
